service-proxy.js

```diff
@@ -827,13 +827,14 @@ function requestReapPeers() {
 ServiceDispatchHandler.prototype.reapPeers =
 function reapPeers() {
     var self = this;
+    self.reapPeersTimer = null;
```

Comment:
This is needed such that the next requestReapPeers goes through after this is triggered by a timeout.